contracts/TokenRequest.sol

```diff
@@ -121,10 +182,15 @@ contract TokenRequest is AragonApp {
         }
 
         addressesTokenRequestIds[msg.sender].deleteItem(_tokenRequestId);
+        uint64 refundedDate = getTimestamp64();
 
-        emit TokenRequestRefunded(refundToAddress, refundToken, refundAmount);
+        emit TokenRequestRefunded(_tokenRequestId, refundToAddress, refundToken, refundAmount, refundedDate);
```

Comment:
There is no need to add `refundedDate` to this Event as we can always get the time the event was emitted from the time of the block it was in. In the script we can get the current block from the `storeEvent` (where we get the event name from) then do something like:
```
const eventBlockTime = (await api.web3Eth('getBlock', blockNumber).toPromise()).timestamp
```